src/components/navTabs/NavTabs.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../styles/base/_variables.scss';
```

Comment:
Not that it's a major thing but it's not a must you include the _ in the file import for scss

Reply:
Thanks for that